src/parsing/parsers/generic/jison/create/create_table.jison

```diff
@@ -52,7 +52,7 @@ TableDefinitionRightPart_EDIT
        keywords.push({ value: 'LIKE', weight: 1 });
      } else {
        if (!$2) {
-         keywords.push({ value: 'PARTITIONED BY', weight: 12 });
+         keywords.push({ value: 'PARTITION BY', weight: 12 });
```

Comment:
Previously we had wrong suggestion value actually, because `PARTITIONED BY` token not exists and this suggestion break syntax